ansible/eos_install_config

```diff
@@ -42,26 +45,39 @@ requirements:
 options:
     hostname:
         description: IP or FQDN of the device you want to connect to
-        required: true
+        required: True
```

Comment:
Perfect, as long as it'll all be the same my OCD will be happy :) converting back to lower case...